guides-and-standards/guide-choosing-software-dependencies.md

```diff
@@ -0,0 +1,59 @@
+# Selecting software dependencies for workflows and software
+
+We often expect that a computational analysis we write at Arcadia will either be used again by another Arcadian or automated for high throughput studies. When this is the case, the software quality used in those analyses becomes more important than if the analysis is a one-off task used to generate a specific biological insight.
+
+## Deciding when your task meets the re-use bar
+
+**When working on a software tool or workflow you anticipate that you or another Arcadian will use multiple times, you need to be careful about which tools you include**. This is _not_ the same as a workflow that documents the steps taken in an analysis project and is often bespoke and specific to a given task. Examples of existing tools or workflows where we should be careful about the quality of tools we use include:
+
+- nf-core-style Nextflow workflows like [NovelTree](https://github.com/Arcadia-Science/noveltree), [PreHGT](https://github.com/Arcadia-Science/prehgt), and [reads2genome](https://github.com/Arcadia-Science/reads2genome)
+- Snakemake workflows written as re-usable tools like [ProteinCartography](https://github.com/Arcadia-Science/ProteinCartography/)
+- Python packages like [plm-utils](https://github.com/Arcadia-Science/2024-plm-utils).
+- Tools designed to analyze experimental data in high throughput like [arcadia-phenotypeomat-protocol]([https://github.com/Arcadia-Science/arcadia-phenotypeomat-protocol)
+
+## Things to look for when deciding to include a tool as a dependency
+
+A dependency is any software tool that is required to make your tool work. For example, BLAST is a dependency of ProteinCartography, and Nextflow and OrthoFinder are dependencies of NovelTree. Below we include some signals for assessing whether a software tool is high quality.
```

Comment:
I get that technically, yes, Nextflow is a dependency of our workflows that use this, but since this sentence starts with the examples of BLAST, I think it's confusing to talk about dependencies as both tools to run steps and the underlying workflow orchestration. Since the rest of this seems to hover around BLAST-like dependencies, I would take out Nextflow from here

Reply:
done!